src/coord/src/catalog.rs

```diff
@@ -2412,9 +2412,16 @@ impl<S: Append> Catalog<S> {
             None => return Ok(()),
         };
         let user = session.user().to_string();
-        let occurred_at = self.state.config.start_time.timestamp_nanos() as u64
-            + self.state.config.start_instant.elapsed().as_nanos() as u64;
```

Comment:
We can stop caring so hard about time ordering correctness now, and just rely on the same clock everything else does for getting correct enough answers to "when did this happen". This assumes all our ec2 machines and pods are correctly using chrony, which I think is fair.